db/schema.rb

```diff
@@ -20,8 +20,8 @@
     t.integer  "user_id"
     t.integer  "recipe_id"
     t.text     "action"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at"
```

Comment:
that happens to me if i run migriations through fig (`fig run web rake db:migrate`). maybe because of pg in the container?